
syntax = "proto2";
package managarm.svrctl;
option optimize_for = LITE_RUNTIME;

enum Error {
	SUCCESS = 0;
	ILLEGAL_REQUEST = 1;
}

enum CntReqType {
	NONE = 0;
	FILE_UPLOAD = 1;
	SVR_RUN = 2;
}

message CntRequest {
	optional CntReqType req_type = 1;
	optional string name = 2;
}

message SvrResponse {
	optional Error error = 1;
}

